database/README.md

```diff
@@ -7,148 +7,98 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/README.
 
-With these settings, you can expect a little `iowait` spike once every 10 minutes and in case of system crash, data on
-disk will be up to 10 minutes old.
+Embedded devices usually have very limited RAM resources available.
 
-![image](https://cloud.githubusercontent.com/assets/2662304/23835030/ba4bf506-0768-11e7-9bc6-3b23e080c69f.png)
+There are 2 settings for you to tweak:
 
-To have these settings automatically applied on boot, create the file `/etc/sysctl.d/netdata-memory.conf` with these
-contents:
+1. `[db].update every`, which controls the data collection frequency
+2. `[db].retention`, which controls the size of the database in memory (except for `[db].mode = dbengine`)
 
-```conf
-vm.dirty_expire_centisecs = 60000
-vm.dirty_background_ratio = 80
-vm.dirty_ratio = 90
-vm.dirty_writeback_centisecs = 0
-```
+By default `[db].update every = 1` and `[db].retention = 3600`. This gives you an hour of data with per second updates.
 
-There is another mode to help overcome the memory size problem. What is **most interesting for this setup** is
-`[db].mode = dbengine`.
+If you set `[db].update every = 2` and `[db].retention = 1800`, you will still have an hour of data, but collected once
+every 2 seconds. This will **cut in half** both CPU and RAM resources consumed by Netdata. Of course experiment a bit.
+On very weak devices you might have to use `[db].update every = 5` and `[db].retention = 720` (still 1 hour of data, but
+1/5 of the CPU and RAM resources).
 
-### dbengine
+You can also disable [data collection plugins](/collectors/README.md) you don't need. Disabling such plugins will also
+free both CPU and RAM resources.
 
-In this mode, the database of Netdata is stored in database files. The [Database Engine](/database/engine/README.md)
-works like a traditional database. There is some amount of RAM dedicated to data caching and indexing and the rest of
-the data reside compressed on disk. The number of history entries is not fixed in this case, but depends on the
-configured disk space and the effective compression ratio of the data stored.
+## Running a dedicated parent Netdata server
 
-We suggest to use **this** mode on nodes that also run other applications. The Database Engine uses direct I/O to avoid
-polluting the OS filesystem caches and does not generate excessive I/O traffic so as to create the minimum possible
-interference with other applications. Using mode `dbengine` we can overcome most memory restrictions. For more
-details see [here](/database/engine/README.md).
+Netdata allows streaming data between Netdata nodes in real-time. This allows you having one or more parent Netdata
+servers that will maintain the entire database for all the nodes that connect to them (their children), and will also
+run health checks/alarms for all these nodes. Explore this feature following the following
+section: [enable streaming between nodes](/docs/metrics-storage-management/enable-streaming.md)
 
 ## KSM
```

Comment:
See https://github.com/rothgar/netdata-wiki/blob/master/Memory-Deduplication---Kernel-Same-Page-Merging---KSM.md and bring in the first few lines in again. We lost some valuable info from omitting those two lines.